modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/portlet/action/EditCPDefinitionMVCActionCommand.java

```diff
@@ -587,6 +608,71 @@ private CPDefinition _updateCPDefinition(
 		return cpDefinition;
 	}
 
+	private CPDefinition _updateCPDefinition(
```

Comment:
@andrea-ale-sbarra , the method itself is absolutely necessary because the existing `_updateCPDefinition` will fail for our use-case. We need to force the update call with the exact same values of the `cpDefinition` and not rely on the request.